rfc/github-org-migration.md

```diff
@@ -0,0 +1,154 @@
+> (...)
+> "All links to the previous repository location are automatically redirected to the new location. When you use git clone, git fetch, or git push on a transferred repository, these commands will redirect to the new repository location or URL. However, to avoid confusion, we strongly recommend updating any existing local clones to point to the new repository URL."
+
+This ensures migration disruptions are minimal and that we can keep operating normally and migrate on a repo-by-repo basis. Still, it is crucial to look at each repo individually to ascertain migration consequences which may impact other systems outside of Github and our local development machines.
+Additionally, it is RECOMMENDED to perform the [post-migration instructions](#post-migration-instructions) below.
+
+CI pipelines SHALL be individually checked to make sure nothing breaks or relies on a particular repo naming structure.
+
+## Repository Overview
+
+**Note: This section is under construction and it is where the bulk of work of this spec will go**
+
+We have a total of 60 candidate repositories to migrate.
+
+The following repos lists were computed with: `curl -Ls -H "Accept: application/vnd.github+json" -H "Authorization: Bearer $GITHUB_TOKEN" -H "X-GitHub-Api-Version: 2022-11-28" https://api.github.com/orgs/$ORG/repos | jq -r '.[] | "- [ ] [" + .name + "](" + .html_url + ")"' | pbcopy`, where `$ORG ∈ {"web3-storage", "filecoin-saturn"}` and where `GITHUB_TOKEN` is a token with access to both orgs. Further reference [here](https://docs.github.com/en/rest/repos/repos?apiVersion=2022-11-28#list-organization-repositories).
+
+### [Web3 Storage](https://github.com/web3-storage)
+
+- [ ] [ipfs-car](https://github.com/web3-storage/ipfs-car)
+- [ ] [web3.storage](https://github.com/web3-storage/web3.storage)
+- [ ] [web3-file](https://github.com/web3-storage/web3-file)
+- [ ] [go-w3s-client](https://github.com/web3-storage/go-w3s-client)
+- [ ] [wrangler-action](https://github.com/web3-storage/wrangler-action)
+- [ ] [add-to-web3](https://github.com/web3-storage/add-to-web3)
+- [ ] [files-from-path](https://github.com/web3-storage/files-from-path)
+- [ ] [multipart-parser](https://github.com/web3-storage/multipart-parser)
+- [ ] [example-image-gallery](https://github.com/web3-storage/example-image-gallery)
+- [ ] [broker](https://github.com/web3-storage/broker)
+- [ ] [web3-schema](https://github.com/web3-storage/web3-schema)
+- [ ] [example-forum-dapp](https://github.com/web3-storage/example-forum-dapp)
+- [ ] [fauna-dump](https://github.com/web3-storage/fauna-dump)
+- [ ] [db-migration-pipeline](https://github.com/web3-storage/db-migration-pipeline)
+- [ ] [fauna-dumpify](https://github.com/web3-storage/fauna-dumpify)
```

Comment:
these can really be archived and dropped from migration as we used them for old DB migrations of old products